client/my-sites/domains/domain-management/components/primary-domain-selector/style.scss

```diff
@@ -0,0 +1,33 @@
+@import "@wordpress/base-styles/breakpoints";
+
+.domains-set-primary-address {
+	&__title {
+		margin-top: 0;
+	}
+	&__subtitle {
+		color: var(--color-text-subtle);
+		margin-bottom: 0;
+		font-size: 0.875rem;
+	}
+	&__fieldset.form-fieldset {
+		margin-top: 0.875rem;
+		margin-bottom: 0;
+	}
+	&__select.form-select {
+		max-width: 100%;
+		width: 100%;
+		@media ( min-width: $break-large ) {
+			width: auto;
+		}
+	}
+	&__submit {
+		margin-right: 0;
+		margin-top: 12px;
+		display: block;
+		@media ( min-width: $break-large ) {
+			display: inline-block;
+			margin-top: 0;
+			margin-left: 12px;
+		}
+	}
```

Comment:
I'm afraid that this doesn't really work if the domain or the button is long, which could happen when the string is translated into other languages.

<img width="745" alt="Screenshot 2024-03-20 at 22 20 29" src="https://github.com/Automattic/wp-calypso/assets/43215253/31f8a98c-3e91-4414-bbd1-024d2f5d5bd0">

What do you think of this?

```suggestion
	&__fieldset.form-fieldset {
		margin-top: 0.875rem;
		margin-bottom: 0;
		@media ( min-width: $break-large ) {
			display: flex;
		}
	}
	&__submit {
		margin: 0;
		margin-top: 12px;
		@media ( min-width: $break-large ) {
			margin: auto;
			margin-left: 12px;
		}
	}
```